src/getCoinCombination.test.js

```diff
@@ -7,4 +7,74 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return an array`, () => {
+    expect(getCoinCombination(1))
+      .toBeInstanceOf(Array);
+  });
+
+  it(`should return an array that lenght = 4`, () => {
+    expect(getCoinCombination(2).length)
+      .toBe(4);
```

Comment:
Use .`toHaveLength` to check that an object has a .`length` property and it is set to a certain numeric value.

Reply:
Fixed.